### PR TITLE
Update Query.md documentation broken link

### PR DESCRIPTION
**Pull request by tristansokol**
The closest page to what I think this used to reference is https://github.com/silverbulletmd/silverbullet/blob/d609daf59ecf5855d0688ceb8f2a10e9f21a14d0/plug-api/lib/query.ts and I'm not sure if that provides much value in terms of understanding any of the query grammar.

### CODE REVIEW

**Comment by zefhemel** [comment]
Actually it moved here: https://github.com/silverbulletmd/silverbullet/blob/main/common/markdown_parser/query.grammar

If you could update that in this PR that'd be nice :)

File: website/🔌 Directive/Query.md [hunk 1]
```diff
@@ -12,9 +12,6 @@ The `#query` is the most widely used directive. It can be used to query various
    * `select`
    * `render`
```
Comment (tristansokol) [inline website/🔌 Directive/Query.md:14]:
```suggestion

P.S.: If you are a developer or have a technical knowledge to read a code and would like to know more about syntax, please check out
[query grammar](https://github.com/silverbulletmd/silverbullet/blob/main/common/markdown_parser/query.grammar).
```

File: website/🔌 Directive/Query.md [hunk 2]
```diff
@@ -13,8 +13,7 @@ The `#query` is the most widely used directive. It can be used to query various
...
 
 P.S.: If you are a developer or have a technical knowledge to read a code and would like to know more about syntax, please check out
-[query grammar](https://github.com/silverbulletmd/silverbullet/blob/main/plugs/directive/query.grammar).
-
+[query grammar](https://github.com/silverbulletmd/silverbullet/blob/main/common/markdown_parser/query.grammar).
```
Comment (tristansokol) [inline website/🔌 Directive/Query.md:16]:
```suggestion
[query grammar](https://github.com/silverbulletmd/silverbullet/blob/main/common/markdown_parser/query.grammar).

```

**Comment by tristansokol** [comment]
> Actually it moved here: https://github.com/silverbulletmd/silverbullet/blob/main/common/markdown_parser/query.grammar
> 
> If you could update that in this PR that'd be nice :)

done!

**Pull request merged**